pennylane/__init__.py

```diff
@@ -28,6 +28,7 @@
 import pennylane.init
 import pennylane.templates.layers
 import pennylane.templates.embeddings
+import pennylane.templates.subroutines
```

Comment:
Will all layers, embeddings, and subroutines be imported here, so that you can do `qml.templates.StronglyEntangling` etc?